google/cloud/spanner/samples/samples.cc

```diff
@@ -3802,6 +3802,16 @@ void QueryInformationSchemaDatabaseOptions(
 }
 // [END spanner_query_information_schema_database_options]
 
+// [START spanner_set_batching_delay]
+void CommitWithBatchingDelay(google::cloud::spanner::Client client,
```

Comment:
The samples should be executed, but no one calls this.

Reply:
Done.